<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoDataForm</class>
 <widget class="QWidget" name="DoDataForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>418</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>746</width>
    <height>418</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>746</width>
    <height>418</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelOtrasl">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Отрасль:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRayon">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>27</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>312</width>
     <height>27</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Район:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbRayon">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>471</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbOtrasl">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>471</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>3</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>731</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>731</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="labelProfs">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>241</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Численность членов профсоюза</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelWorkers">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Численность работников</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbWorkers">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::PlusMinus</enum>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelPredName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>102</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Наименование организации</string>
   </property>
   <property name="buddy">
    <cstring>EditPredName</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditPredName">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>511</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>731</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>361</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="title">
    <string>Руководитель организации</string>
   </property>
   <widget class="QLabel" name="labelWPosition">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>31</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Должность</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelWFIO">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>61</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ФИО</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelWTel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>91</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Телефон</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditWPosition">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>251</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditWFIO">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>251</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditWTel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>251</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>200</y>
     <width>361</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Руководитель профсоюзной организации</string>
   </property>
   <widget class="QLineEdit" name="EditPTel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>251</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditPPosition">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>251</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditPFIO">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>251</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelPTel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>91</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Телефон</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPPosition">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>31</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Должность</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPFIO">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ФИО</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>731</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Адрес</string>
   </property>
   <widget class="QLineEdit" name="EditAdress">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>711</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QSpinBox" name="sbProfs">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>150</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="showGroupSeparator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
